src/composables/trade/useSor.ts

```diff
@@ -496,7 +496,7 @@ export default function useSor({
       onTxConfirmed: () => {
         trading.value = false;
         latestTxHash.value = tx.hash;
-        trackGoal(Goals.Swapped);
+        trackGoal(Goals.Swapped, Number(tokenInAmountInput.value) || 0);
```

Comment:
Is there a way to add the token symbol or USD value here? as the token amount by itself isn't a very useful metric .